### PR TITLE
Navbar image

### PR DESCRIPTION
**Pull request by scottcrawford03**
Navbar now shows current user image.

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <header class='mod-header'>
   <div class='layout-wrap'>
     <h1><a href="http://turing.io">Turing School of Software & Design</a></h1>
+      <%= image_tag(current_user.profile_image_url, class: "nav-image img-circle", width: '7%', height: '7%') %>
```
Comment (larsonkonr) [inline app/views/layouts/_navbar.html.erb:4]: should the style be pulled out to the css

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -3,11 +3,19 @@
...
-  </div>
-  <%= link_to "/auth/github" do %>
-  <div class="login-button">
-    LOGIN
+    <%= image_tag('turing-logo-copy.png', width: '15%', height: '15%') %>
   </div>
+  <% if current_user.nil? %>
+    <%= link_to "/auth/github" do %>
+      <div class="login-button">
+        LOGIN
```
Comment (larsonkonr) [inline app/views/home/index.html.erb:11]: ALL CAPS BRUH!

Comment (scottcrawford03) [inline app/views/home/index.html.erb:11]: :yellow_heart:

File: app/views/layouts/_navbar.html.erb (hunk 1 above)
Comment (scottcrawford03) [inline app/views/layouts/_navbar.html.erb:4]: I think in this case it makes sense on the inline. :ice_cream:

**Pull request merged**